projects.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python
+
+import argparse
+import logging
+from pathlib import Path
+import time
+import json
+
+from dcctools.anonlink import Project
+from dcctools.config import Configuration
+
+# Delay between run status checks
+SLEEP_TIME = 10.0
+
+log = logging.getLogger(__name__)
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--config",
+        default="config.json",
+        help='Configuration file (default "config.json")',
+    )
+    parser.add_argument(
+        "--verbose", default=False, action="store_true", help="Show debugging output"
+    )
+    return parser.parse_args()
+
+def run_projects(c):
+    if c.household_match:
+        log.debug("Processing households")
+        with open(Path(c.household_schema)) as schema_file:
+            household_schema = schema_file.read()
+            project_name = "fn-phone-addr-zip"
+            household_project = Project(
+                project_name,
+                household_schema,
+                c.systems,
+                c.entity_service_url,
+                c.blocked,
+            )
+            household_project.start_project()
+            for system in c.systems:
+                household_project.upload_clks(
+                    system, c.get_household_clks_raw(system, project_name)
+                )
+
+            if type(c.matching_threshold) == list:
+                threshold = c.matching_threshold[0]
+            else:
+                threshold = c.matching_threshold
+
+            household_project.start_run(threshold)
+            household_project.start_run(c.matching_threshold)
```

Comment:
I think we missed something in Jonathan's last PR -- this bit should use the `household_threshold` property from the config.json. Also I think the double start_run might be a merge issue

 Other than that everything looks good to merge